### PR TITLE
bad coding

### PR DESCRIPTION
**Pull request by LuigiImVector**
no

### CODE REVIEW

File: .github/workflows/fpb-lint.yml [hunk 1]
```diff
@@ -14,7 +19,26 @@ jobs:
...
+    - run: |
+        fpb-lint ./casts/ 2>> error-log.txt | echo error-log.txt
+    - run: |
+        fpb-lint ./courses/ 2>> error-log.txt | echo error-log.txt
+    - run: | 
+        fpb-lint ./more/ 2>> error-log.txt | echo error-log.txt
+
+    - run: |
+        gh pr review $PR -a
+        if: success() &&
```
Comment (davorpa) [inline .github/workflows/fpb-lint.yml]: Take care about indentations into YAML config files :wink:

image

File: .github/workflows/fpb-lint.yml [hunk 2]
```diff
@@ -14,7 +19,26 @@ jobs:
       with:
         node-version: '16.x'
     - run: npm install -g free-programming-books-lint
-    - run: fpb-lint ./books/
-    - run: fpb-lint ./casts/
-    - run: fpb-lint ./courses/
-    - run: fpb-lint ./more/
+    - run: |
+        fpb-lint ./books/ 2>> error-log.txt | echo error-log.txt
```
Comment (davorpa) [inline .github/workflows/fpb-lint.yml]: Has you tried `tee` command like Seth does in his workflow?  

https://github.com/SethFalco/free-programming-books/blob/e9f7dff3be87e640a72f97bd4eec1be79ec2b43a/.github/workflows/fpb-lint.yml#L17

I think it's more apropiate to print and conserve content of stdout stream 

In this case append modifier is needed. See man... https://man7.org/linux/man-pages/man1/tee.1.html

```shell
fpb-lint [folder] 2>&1 | tee -a output.log
```

Comment (LuigiImVector) [reply .github/workflows/fpb-lint.yml]: using `tee` github didn't understand the errors

Comment (LuigiImVector) [reply .github/workflows/fpb-lint.yml]: `tee` output: /home/runner/work/free-programming-books/free-programming-books/books/free-programming-books-langs.md

File: .github/workflows/fpb-lint.yml (hunk 1 above)
Comment (davorpa) [reply .github/workflows/fpb-lint.yml]: Said that...

```suggestion
      if: success() &&
```

solves coding error, I thought

Comment (LuigiImVector) [reply .github/workflows/fpb-lint.yml]: fixed by creating a step

File: .github/workflows/fpb-lint.yml [hunk 3]
```diff
@@ -14,7 +19,25 @@ jobs:
...
-    - run: fpb-lint ./courses/
-    - run: fpb-lint ./more/
+    - run: |
+        fpb-lint ./books/ &>> output.log || cat output.log
+    - run: |
+        fpb-lint ./casts/ &>> output.log || cat output.log
+    - run: |
+        fpb-lint ./courses/ &>> output.log || cat output.log
+    - run: | 
+        fpb-lint ./more/ &>> output.log || cat output.log
```
Comment (davorpa) [inline .github/workflows/fpb-lint.yml:29]: `&>> output.log` is equivalent to `>> output.log 2>&1` or `>> output.log 2>> output.log`, so homogenize

If you want batch linters.... the problem, I thought, is that `run` executions are exited inmediatelly if some error is raised.

Try...
```yml
- run: fpb-lint [folder] &>> output.log || cat output.log
  if: always()
```

**Pull request closed without merging**